src/main/java/Wqchat.java

```diff
@@ -0,0 +1,113 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+public class Wqchat {
+    public static void printLine() {
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void printTaskCount() {
+        if (taskCount == 0) {
+            System.out.println("Now you have 1 task in the list.");
+        } else {
+            int count = taskCount + 1;
+            System.out.println("Now you have " + count + " tasks in the list");
+        }
+    }
+    protected static int taskCount = 0;
+    public static void main(String[] args) {
+        /*ArrayList<Task> tasks = new ArrayList<>(); // a list of tasks*/
+        Task[] tasks = new Task[100];
+
+
+        printLine();
+        System.out.println("Hello! I'm Wqchat");
+        System.out.println("What can I do for you?");
+        printLine();
+
+        String line;
+        Scanner in = new Scanner(System.in);
+        line = in.nextLine();
+
+        while (!line.equals("bye")) {
+            if (line.equals("list")) {
```

Comment:
The main() function seems unnecessarily long as it contains multiple nested if-else blocks, which can make it hard to read and maintain. Consider breaking it down into smaller, more modular methods.

Reply:
There is a lot of repeated code within the if-else blocks. Consider also refactoring common functionality into separate methods to reduce duplication

Reply:
You could also consider using switch cases in place of multiple if-else blocks for better readability